fontforge/splinefont.h

```diff
@@ -1931,7 +1931,7 @@ typedef struct splinefont {
     int woffMajor;
 #define woffUnset		0x4455
     int woffMinor;
-    char *woffMetadata;
```

Comment:
Or, rather, this is splinefont.h. But why redefine it here (and inevitably elsewhere) when you can just cast the interactions with uncompress?